gap/oper.gi

```diff
@@ -1684,3 +1684,53 @@ function(digraph, list)
 
   return DigraphShortestDistance(digraph, list[1], list[2]);
 end);
+
+# Partial Order Digraphs
+
+InstallMethod(PartialOrderDigraphJoinOfVertices,
+"for a digraph and two positive integers",
+[IsDigraph, IsPosInt, IsPosInt],
+function(digraph, i, j)
+  local x, nbs, intr;
+
+  if not IsPartialOrderDigraph(digraph) then
+    ErrorNoReturn("Digraphs: PartialOrderDigraphJoinOfVertices: usage, \n",
+                  "the first argument must be a partial order digraph,");
```

Comment:
it's often helpful to give the problematic variable name so that the user can debug if necessary. So if this instead said

```
...first argument <digraph> must be...`
```
Then the user knows that they can access the digraph with the variable name digraph.